.github/workflows/run-tests.yml

```diff
@@ -237,7 +237,7 @@ jobs:
         env:
           ALLURE_RESULTS_DIR: ${{ env.ALLURE_RESULTS_DIR }}
         run: |
-          source venv.pytest/bin/activate && pytest --alluredir=${GITHUB_WORKSPACE}/allure-results pytest_tests/tests
+          source venv.pytest/bin/activate && pytest --log-cli-level error --alluredir=${GITHUB_WORKSPACE}/allure-results pytest_tests/tests
```

Comment:
Why?

Reply:
We don't need all of the logs when running in CI. They make it harder to see actual errors. Besides, we regularly have allure, that makes these logs even more useless in CI. When running locally - it is a different story and the log level is INFO there.

Reply:
Why is it not explained in the commit message?

Reply:
Updated the commit message.